src/main/cpp/chassis/driveStates/DriveToNote.cpp

```diff
@@ -68,21 +69,26 @@ pathplanner::PathPlannerTrajectory DriveToNote::CreateDriveToNote()
     auto visiondata = DragonVision::GetDragonVision();
     auto notedata = visiondata->GetVisionData(DragonVision::VISION_ELEMENT::NOTE);
 
+    auto finder = DragonDriveTargetFinder::GetInstance();
+
+    auto info = finder->GetPose(DragonVision::VISION_ELEMENT::NOTE);
+    auto type = get<0>(info);
+    auto targetNotePose = get<1>(info);
+
     frc::DriverStation::Alliance allianceColor = FMSData::GetInstance()->GetAllianceColor();
 
     pathplanner::PathPlannerTrajectory trajectory;
 
-    if (visiondata != nullptr && m_chassis != nullptr)
+    if (type != DragonDriveTargetFinder::TARGET_INFO::NOT_FOUND && m_chassis != nullptr)
     {
         auto currentPose2d = m_chassis->GetPose();
 
         if (allianceColor == frc::DriverStation::kBlue)
         {
-            units::angle::degree_t currentnotedirection = dtnvisiondata->getNoteDirection();
```

Comment:
both branches of this appear to be the same, so we don't need to check alliance color and do processing for each separately.

Seems like we would just get the current pose (from chassis) and the note pose (from auto finder = DragonDriveTargetFinder::GetInstance();

    auto info = finder->GetPose(DragonVision::VISION_ELEMENT::NOTE);) and then create a path between them .   I'm not even sure in this case we need to add an extra point.